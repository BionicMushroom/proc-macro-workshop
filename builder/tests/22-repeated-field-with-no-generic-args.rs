use derive_builder::Builder;

struct Vec;

#[derive(Builder)]
struct Test {
    #[builder(each = "whatever")]
    field: Vec,
}

fn main() {}
