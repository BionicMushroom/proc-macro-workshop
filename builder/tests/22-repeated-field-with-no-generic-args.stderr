error: expected standard Vec
 --> tests/22-repeated-field-with-no-generic-args.rs:8:12
  |
8 |     field: Vec,
  |            ^^^
